### ISSUE TITLE
Manifesto for Software Craftsmanship

### ISSUE THREAD

**Issue by pyrige** [dwmkerr/hacker-laws#187]
I think the **Manifesto for Software Craftsmanship** would make a decent addition, even if I wouldn't exactly know where to put it. It also already comes with a bunch of translations.

>  As aspiring Software Craftsmen we are raising the bar of professional software development by practicing it and helping others learn the craft. Through this work we have come to value:
>
> Not only working software,
but also **well-crafted software**
Not only responding to change,
but also **steadily adding value**
Not only individuals and interactions,
but also a **community of professionals**
Not only customer collaboration,
but also **productive partnerships**
>
> That is, in pursuit of the items on the left we have found the items on the right to be indispensable.

**Comment by dwmkerr** [comment]
Yep makes sense - a bit like the Agile Manifesto and the Zen of Python, or the 12 factor app, collections of principles are a good thing to add (see also #180)

**Comment by umutphp** [comment]
Hi @dwmkerr ,
A new chapter/section with name "Manifestos" can be added to the article for such kind of contents.
